container/split.go

```diff
@@ -20,6 +20,11 @@ type Split struct {
 	Horizontal bool
 	Leading    fyne.CanvasObject
 	Trailing   fyne.CanvasObject
+
+	// to communicate to the renderer that the next refresh
+	// is just an offset update (ie a resize and move only)
+	// cleared by renderer in Refresh()
+	nextRefreshIsOffsetUpdate bool
```

Comment:
This is a bit verbose. Wouldn’t something like `offsetUpdated` suffice?

Reply:
Sounds fine. Updated :)